src/controllers/mouseoverTextController.js

```diff
@@ -1,43 +1,48 @@
-const mouseoverTextController = (function (MouseoverText) {
-    const createMouseoverText = function (req, res) {
-        const newMouseoverText = new MouseoverText();
-        newMouseoverText.mouseoverText = req.body.newMouseoverText;
-        newMouseoverText.save().then(() => res.status(201).json({
-            _serverMessage: 'MouseoverText succesfuly created!',
-            mouseoverText: newMouseoverText,
-        })).catch(err => res.status(500).send({ err }));
-    };
+const mouseoverTextController = function (MouseoverText) {
+  const createMouseoverText = function (req, res) {
+    const newMouseoverText = new MouseoverText();
+    newMouseoverText.mouseoverText = req.body.newMouseoverText;
+    newMouseoverText
+      .save()
+      .then((result) =>
+        res
+          .status(201)
+          .send({ _serverMessage: 'MouseoverText successfully created!', mouseoverText: result }),
+      )
+      .catch((error) => res.status(500).send(error));
```

Comment:
Before it was sending the error enclosed in an object: `{ err }`. Now, you are sending the error as is. Have you tested this won't break the frontend app?